server/src/socket/handlers/handleMessage.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose';
+
+import Messages from '../../models/messages';
+import { messageEvent } from '../../constants/eventNames';
+
+const handleMessage = socket => async ({ channelId, message }) => {
+  const senderId = socket.store.user._id;
+  const messageData = {
+    message,
+    read: false,
+    channelId: mongoose.Types.ObjectId(channelId),
+    sender: mongoose.Types.ObjectId(senderId),
+    timestamp: new Date(),
```

Comment:
Instead of adding a timestamp field you can use the timestamp option mongoose provide, it can automatically add createdAt and updatedAt field to the collection.

Reply:
Yes right.

Right now I think API code is also using `timestamp` field. Could I create another PR to remove timestamp field everywhere and use timestamp option.